### DISCUSSION TITLE
Using integration Pocket LAN

### DISCUSSION THREAD

**Discussion by howard-paster** [wills106/homeassistant-solax-modbus#947] [Q&A]
No mention of how this should/could work. Obviously no need for Modbus/TCP/Ethernet conversion as have native IP connection to network. Is this even possible. Just want to build Octopus/Solax connection

**Comment by wills106** [comment]
The older Pocket LAN's don't provide a Modbus connection, so can't be used.
https://homeassistant-solax-modbus.readthedocs.io/en/latest/compatible-adaptors/#solax-pocketlan

**Comment by howard-paster** [comment]
The Pocket LAN is the latest only 12 months old, but obviously only
provided a LAN/Ethernet connection no WiFi
Does this mean there is no HA integration available either by you or
someone else

On Wed, 3 Jul 2024 at 08:07, wills106 ***@***.***> wrote:

> The older Pocket LAN's don't provide a Modbus connection, so can't be used.
>
> https://homeassistant-solax-modbus.readthedocs.io/en/latest/compatible-adaptors/#solax-pocketlan
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wills106/homeassistant-solax-modbus/discussions/947#discussioncomment-9944521>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJS5UA3TFNKFCZMJRNDM6VLZKOPJRAVCNFSM6AAAAABKIY6PP2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSNBUGUZDC>
> .
> You are receiving this because you authored the thread.Message ID:
> <wills106/homeassistant-solax-modbus/repo-discussions/947/comments/9944521
> @github.com>
>


-- 
*Howard*

**Comment by howard-paster** [comment]
So we have a pocket lan v3 dated July 2022
I have logged into inverter and it seems to have a Modbus address
[image: image.png]

Configuring in HomeAssistant
[image: image.png]
[image: image.png]
Works but no devices show
[image: image.png]

I don't want to go down rabbit hole - but struggling to work out if this is
possible with my setup
Frustrated if answer is no - or requires major hardware update



On Wed, 3 Jul 2024 at 08:44, Howard Paster ***@***.***> wrote:

> The Pocket LAN is the latest only 12 months old, but obviously only
> provided a LAN/Ethernet connection no WiFi
> Does this mean there is no HA integration available either by you or
> someone else
>
> On Wed, 3 Jul 2024 at 08:07, wills106 ***@***.***> wrote:
>
>> The older Pocket LAN's don't provide a Modbus connection, so can't be
>> used.
>>
>> https://homeassistant-solax-modbus.readthedocs.io/en/latest/compatible-adaptors/#solax-pocketlan
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/wills106/homeassistant-solax-modbus/discussions/947#discussioncomment-9944521>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BJS5UA3TFNKFCZMJRNDM6VLZKOPJRAVCNFSM6AAAAABKIY6PP2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSNBUGUZDC>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <wills106/homeassistant-solax-modbus/repo-discussions/947/comments/9944521
>> @github.com>
>>
>
>
> --
> *Howard*
>


-- 
*Howard*
